cache/s3.go

```diff
@@ -0,0 +1,173 @@
+package cache
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"github.com/aquasecurity/fanal/types"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/s3"
+	"github.com/aws/aws-sdk-go/service/s3/s3iface"
+	"github.com/aws/aws-sdk-go/service/s3/s3manager"
+	"github.com/google/go-containerregistry/pkg/v1"
+	"golang.org/x/xerrors"
+)
+
+type S3Cache struct {
+	S3         s3iface.S3API
+	Downloader *s3manager.Downloader
+	BucketName string
```

Comment:
Is there any reason to export these variables?

```suggestion
	s3         s3iface.S3API
	downloader *s3manager.Downloader
	bucketName string
```